util/cron/test-gpu-ex-cuda-12.interop.bash

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+#
+# GPU native testing on a Cray EX (using none for CHPL_COMM)
+
+CWD=$(cd $(dirname ${BASH_SOURCE[0]}) ; pwd)
+source $CWD/common.bash
+source $CWD/common-hpe-cray-ex.bash
+
+
+# We need 12.4 for the stream test because the CUDA driver on pinoak
+# only supports PTX for 12.4, untill the driver is updated, we need to
+# stick with 12.4 instead of 12.5
+module load cuda/12.4  # default is CUDA 12
+
+# We need cublas for the cublas interop test, but since we are using 12.4 above
+# pinoak doesn't have the cublas library for 12.4, so we need to use the cublas
+# from 12.5 (which is compatible across minor versions)
+# This can be removed once we use CUDA 12.5
+export CHPL_LIB_PATH="/opt/nvidia/hpc_sdk/Linux_x86_64/24.7/math_libs/lib64"
+
+export CHPL_LLVM=bundled  # CUDA 12 is only supported with bundled LLVM
```

Comment:
Is this comment still applicable? I wouldn't think so. You can still keep the bundled LLVM if you choose to because that's safer (you can replace the comment saying something like that)

Reply:
I'm not sure. @jabraham17 might know the status of what we support with system llvm better.

Reply:
This is no longer accurate, we can use LLVM=system with CUDA 12 for LLVM 16+